src/table/components/virtualized-table/__tests__/TableContainer.spec.tsx

```diff
@@ -191,4 +191,66 @@ describe('<TableContainer />', () => {
       await waitFor(() => expect(getByText('measure-c2-r0').parentNode).toHaveClass('excluded'));
     });
   });
+
+  describe('totals', () => {
+    it('should render with totals at the top when total is in auto mode', async () => {
```

Comment:
I would say this one isn't really needed. The logic in handle-data is covered well, so you should only need to test the three different cases it could render

Reply:
At this level I dont want to assume to much about what lower level functions are being used.

I do however think it would be great if I could verify the actual position and not used that the totals have rendered.